.expeditor/verify.pipeline.yml

```diff
@@ -577,6 +577,20 @@ steps:
       automatic:
         limit: 1
 
+  - label: "[build] :linux: supplemental-group-tester"
```

Comment:
I missed it; why do we build this here?

Reply:
:thinking: I guess we don't really need to... it makes more sense with the `test-probe` package, since there's actually code to compile there. This is just a dumb service.

I'll remove it; in retrospect, I don't think it really buys us anything.

Reply:
Addressed in https://github.com/habitat-sh/habitat/pull/7823/commits/691da5a033e6221cb442e52f27a5ca880a983bb2